neurodamus/replay.py

```diff
@@ -42,17 +37,10 @@ def open_spike_file(self, filename, delay, population=None):
             filename: path to spike out file. Interpret as binary or ascii according to extension
             delay: delay to apply to spike times
         """
-        # determine if we have binary or ascii file
-        # TODO: filename should be able to handle relative paths,
-        # using the Run.CurrentDir as an initial path
-        # _read_spikes_xxx shall return numpy arrays
         if filename.endswith(".h5"):
```

Comment:
```suggestion
        if Path(filename).suffix == ".h5":
```

so that we move to using paths instead of strings... dunno if you want to add it. I am just presenting an option

Reply:
Good point. Thanks.  I've just changed that.